src/ddscxx/include/org/eclipse/cyclonedds/topic/qos/TopicQosDelegate.hpp

```diff
@@ -54,6 +54,7 @@ class OMG_DDS_API TopicQosDelegate
     void policy(const dds::core::policy::TransportPriority&  priority);
     void policy(const dds::core::policy::Lifespan&           lifespan);
     void policy(const dds::core::policy::Ownership&          ownership);
+    void policy(const dds::core::policy::DataRepresentation& datarepresentation);
```

Comment:
Please also encapsulate this line with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro.